### ISSUE TITLE
Fig not displaying auto complete on full screen iTerm2 on macOS Ventura 

### ISSUE THREAD

**Issue by archiekd** [withfig/fig#1989]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.0 (22A380)

### Expected behaviour

When I have iTerm in fullscreen fig should show autocomplete as I type as it does when I am not in full screen.

### Actual behaviour

When I have iTerm not in full screen fig autocomplete works when I am not in full screen however when I make the window fullscreen I lose all fig overlay visibility.

How it looks when in full screen
<img width="715" alt="Screenshot 2022-11-04 at 18 33 20" src="https://user-images.githubusercontent.com/9201058/200049988-bf27671c-b9dd-49e3-9877-61e2b099942d.png">

How it looks when in not in full screen
<img width="439" alt="Screenshot 2022-11-04 at 18 32 50" src="https://user-images.githubusercontent.com/9201058/200049989-87b6ece4-d0be-4166-a6d8-b55125f4877e.png">

### Steps to reproduce

1. Open iTerm
2. Try when not in fullscreen
3. Put iTerm into full screen
4. Autocomplete stops showing up

### Environment

```yaml
fig-details:
  - 2.7.4
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Max
  - cores: 10
  - mem: 64.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/archiekd/Developer/searchland/nx-monorepo
  - exe-path: /Users/archiekd/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: a6e3fe52-09ed-4052-bce6-bf4266e69947
    - FIG_HOSTNAME: Archies-MacBook-Pro.local
    - FIG_TERM: 2.7.4
    - PATH: /Users/archiekd/Developer/google-cloud-sdk/bin:/Users/archiekd/.pyenv/plugins/pyenv-virtualenv/shims:/Users/archiekd/.pyenv/shims:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/archiekd/.pyenv/bin:/Users/archiekd/.nvm/versions/node/v16.13.0/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Library/Apple/usr/bin:/Users/archiekd/.fig/bin:/Users/archiekd/.local/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 5656
    - FIG_PARENT: a6e3fe52-09ed-4052-bce6-bf4266e69947
    - FIG_SET_PARENT: a6e3fe52-09ed-4052-bce6-bf4266e69947
```

**Comment by ZedMuir** [comment]
I can confirm that I have the same issue. It is however, not just with iTerm; the same bug is occurring with the inbuilt Terminal.app and with the terminal window in VS Code. All three terminals work when the application is not in fullscreen; all three applications fail to show the fig overlay when in fullscreen. 

fig-details:
  - 2.7.4
hardware-info:
  - model:
  - model-id:
  - chip-id: Apple M1 Max
  - cores: 10
  - mem: 32.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/zedmuir
  - exe-path: /Users/zedmuir/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: c70adf16-cbf2-47fb-b6d3-5bccb373379e
    - FIG_HOSTNAME: ZedMBPMax.local
    - FIG_TERM: 2.7.4
    - PATH: /Users/zedmuir/.local/share/fig/plugins/git-open:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/zedmuir/.zplug/bin:/opt/homebrew/opt/ruby/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Applications/Little Snitch.app/Contents/Components:/Users/zedmuir/.fig/bin:/Users/zedmuir/.local/bin:/Users/zedmuir/.fzf/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 1699
    - FIG_PARENT: c70adf16-cbf2-47fb-b6d3-5bccb373379e
    - FIG_SET_PARENT: c70adf16-cbf2-47fb-b6d3-5bccb373379e

**Comment by tonykay** [comment]
Ah! This has been killing me. Never thought to put iTerm out of full-screen mode. I have the same issue. Ventura 13.1 (beta)

**Comment by tonykay** [comment]
Is this a duplicate of`[ issue: 1964](https://github.com/withfig/fig/issues/1964)`?

**Comment by wxh06** [comment]
As @tonykay mentioned, this is probably a duplicate of #1964

**Comment by grant0417** [comment]
This should be fixed on 2.7.7, make sure to update and let us know if the issue is fixed!

**Comment by ZedMuir** [comment]
Fixed for me. Completely fixed for iTerm, Terminal and VSCode terminal instances. Thanks for the great work.
